frontend/src/components/auth/WithAuth.js

```diff
@@ -13,11 +31,12 @@ const withAuth = WrappedComponent => {
     constructor(props) {
       super(props)
       this.state = {
-        verified: false
+        verified: null
       }
     }
 
     async componentDidMount() {
+      this.props.beginLoading()
```

Comment:
wait where is this coming from? Since you're not mapping anything to props and you're importing, but this implies that it's coming from props so I'm a bit confused

Reply:
jk im blind